compiler/ras/CallStack.cpp

```diff
@@ -260,12 +260,18 @@ void TR_LinuxCallStackIterator::printStackBacktrace(TR::Compilation *comp)
 #include <unistd.h>
 #include <ceeedcct.h>
 
+#if !defined(__open_xl__)
 extern "builtin" void *__gdsa();
+#endif // !defined(__open_xl__)
 
 TR_MvsCallStackIterator::TR_MvsCallStackIterator ()
       : TR_CallStackIterator()
    {
+#if defined(__open_xl__)
+   _parms.__tf_dsa_addr = (void*)__builtin_s390_gdsa();
+#else
```

Comment:
```suggestion
#else /* !defined(__open_xl__) */
```

Reply:
I kept this and the below `defined(__open_xl__)` to reflect the original if clause, but replaced with block comments.